src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,936 @@
+1. Transaction `B` writes to row `R` in table `T` and commits at timestamp `2`.
+1. Transaction `A` writes to row `R` in table `T` and commits at timestamp `3`.
+
+The value of `R` has changed while transaction `A` is open. However, `A` can still write to `R` and commit, effectively overwriting the update from transaction `B`.
+
+{{site.data.alerts.callout_info}}
+Under `SERIALIZABLE` isolation, transaction `A` would have aborted with a [`RETRY_WRITE_TOO_OLD`]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}#retry_write_too_old) error, prompting the client to retry the transaction.
+{{site.data.alerts.end}}
+
+##### Example: Lost update anomaly
+
+  <div class="grid-container">
+  <h5 style="text-align:center">Session 1</h5><h5 style="text-align:center">Session 2</h5>
+  <div class="grid-item">
+  In a terminal window (Session 1), create a table and insert some values:
+
+  {% include_cached copy-clipboard.html %}
+  ~~~ sql
+  CREATE TABLE kv (k INT PRIMARY KEY, v INT);
+  ~~~
+
+  {% include_cached copy-clipboard.html %}
+  ~~~ sql
+  INSERT INTO kv VALUES (1, 2);
+  ~~~
+  
+  Begin a `READ COMMITTED` transaction and read a table row:
+
+  {% include_cached copy-clipboard.html %}
+  ~~~ sql
+  BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
+    SELECT * FROM kv WHERE k = 1;
+  ~~~
+  
+  ~~~
+    k | v
+  ----+----
+    1 | 2
+  ~~~
+  </div>
+
+  <div class="grid-item">
+  In a new terminal window (Session 2), begin another `READ COMMITTED` transaction:
+
+  {% include_cached copy-clipboard.html %}
+  ~~~ sql
+  BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
+  ~~~
+  
+  Update the table row and commit the transaction:
+  
+  {% include_cached copy-clipboard.html %}
+  ~~~ sql
+  UPDATE kv SET v = 3 WHERE k = 1;
+    COMMIT;
+  ~~~
+  </div>
+
+  <div class="grid-item">
+  In Session 1, update the table row again and commit the transaction:
+
+  {% include_cached copy-clipboard.html %}
+  ~~~ sql
+  UPDATE kv SET v = 4 WHERE k = 1;
+    COMMIT;
+  ~~~
+  
+  Read the table row and see that it reflects the update from Session 1:
+  
+  {% include_cached copy-clipboard.html %}
+  ~~~ sql
+  SELECT * FROM kv WHERE k = 1;
+  ~~~
+  
+  ~~~
+    k | v
+  ----+----
+    1 | 4
+  ~~~
+  </div>
+  </div>
+
+The update in Session 2 appears to be "lost" because its result is overwritten by a concurrent transaction. It is **not** lost at the database level, and can be found using [`AS OF SYSTEM TIME`]({% link {{ page.version.version }}/as-of-system-time.md %}) and a timestamp earlier than the commit in Session 1:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM kv AS OF SYSTEM TIME '2023-11-09 21:22:10' WHERE k = 1;
+~~~
+
+~~~
+  k | v
+----+----
+  1 | 3
+~~~
+
+{{site.data.alerts.callout_info}}
+While concurrent `READ COMMITTED` transactions can have their writes overwritten, **uncommitted** writes in `READ COMMITTED` transactions cannot be overwritten.
```

Comment:
micro-nit: Maybe just for emphasis, qualify "can have their writes overwritten, ..." as "can have their *committed* writes overwritten, ..."?